app/components/candidate_interface/english_foreign_language/summary_review_component.rb

```diff
@@ -0,0 +1,97 @@
+class CandidateInterface::EnglishForeignLanguage::SummaryReviewComponent < ViewComponent::Base
+  attr_reader :application_form
+
+  def initialize(application_form:)
+    @application_form = application_form
+  end
+
```

Comment:
```suggestion
def call
    render SummaryCardComponent.new(rows: efl_rows, editable: false) do
      render SummaryCardHeaderComponent.new(title: t('.subtitle'), heading_level: 3)
    end
  end
```

Did you know that for simple views like this we can render the markup from within the component by implementing the [`#call` method](https://viewcomponent.org/guide/templates.html#call) this way we don't need a separate `.html.erb` file 🚀

Reply:
ooh cheers, have updated to use that! 🥳